### PR TITLE
add option to run local operators

### PR DESCRIPTION
**Pull request by maelk**
Instead of using 'make deploy' for Baremetal Operator and CAPI
provider baremetal, add an option to use 'make run' and log the
output.

### CODE REVIEW

**Comment by maelk** [comment]
Add a check skip, if the operators are running locally, we should not verify the number of replicas running in the kubernetes cluster (they should be 0)

File: 03_launch_mgmt_cluster.sh [hunk 1]
```diff
@@ -18,6 +19,9 @@ CAPBMREPO="${CAPBMREPO:-https://github.com/metal3-io/cluster-api-provider-bareme
 CAPBMBRANCH="${CAPBMBRANCH:-master}"
 FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
 
+BMO_RUN_LOCAL="${BMO_RUN_LOCAL:-false}"
+CAPBM_RUN_LOCAL="${CAPBM_RUN_LOCAL:-false}"
```
Comment (russellb) [inline 03_launch_mgmt_cluster.sh]: Can you add docs for these new variables to `config_user.sh` ?

Comment (maelk) [reply 03_launch_mgmt_cluster.sh]: done. I also added a test to check that they are running locally

**Comment by russellb** [comment]
This is working for me.  I needed operator-sdk installed, so I added that to our requirements install script too.

**Pull request merged**